diff --git a/node_modules/svelte/compiler.js b/node_modules/svelte/compiler.js
index 5ed2bc6..a133567 100644
--- a/node_modules/svelte/compiler.js
+++ b/node_modules/svelte/compiler.js
@@ -13798,7 +13798,7 @@
 	                value: parent
 	            }
 	        });
-	        this.can_use_innerhtml = !renderer.options.hydratable;
+	        this.can_use_innerhtml = process.env.TARO_ENV === 'h5' ? !renderer.options.hydratable : false;
 	        this.is_static_content = !renderer.options.hydratable;
 	        block.wrappers.push(this);
 	    }
